### ISSUE TITLE
How can I train my model on multiple servers with pytorch?

### ISSUE THREAD

**Issue by DragonBao-ai** [NVIDIA/DALI#1444]
I had test this code
"https://github.com/NVIDIA/DALI/blob/master/docs/examples/pytorch/resnet50/main.py"
with 
"python -m torch.distributed.launch --nproc_per_node=NUM_GPUS main.py -a resnet50 --dali_cpu --fp16 --b 128 --static-loss-scale 128.0 --workers 4 --lr=0.4 ./"

and it works well and fast.Great!

I have 2 servers ,each of them has 8 gpus。
I want to do distributed training using these 16 gpus,so how I can do this.
Is there some example code?

**Comment by JanuszL** [comment]
Hi,
From DALI point of view single or multinode training is transparent. I think it would be best to ask this question where we provides our DL examples - https://github.com/NVIDIA/DeepLearningExamples.
Some kind of reference our [MLPerf benchmark](https://github.com/mlperf/training_results_v0.6/tree/master/NVIDIA/benchmarks/ssd/implementations/pytorch) submission code could be considered. However there are certain assumption about available HW and SW that may not apply in your case.

**Comment by JanuszL** [comment]
This is full example for multinode BART in PyTorch - https://github.com/NVIDIA/DeepLearningExamples/tree/master/PyTorch/LanguageModeling/BERT#multi-node.

**Comment by DragonBao-ai** [comment]
Thank you!
